esbuild.mjs

```diff
@@ -0,0 +1,18 @@
+import {common} from '@shelf/esbuild-config';
+
+await Promise.all(
+  ['esm', 'cjs'].map(
+    async format =>
+      await common({
+        entryPoints: ['src/index.ts'],
+        config: {
+          splitting: false,
```

Comment:
is there a reason why we passing **false** to `splitting`

Reply:
<img width="653" alt="CleanShot 2023-07-06 at 14 24 41@2x" src="https://github.com/shelfio/bem/assets/33541009/efbe867b-127d-4d57-9283-b99070629a0e">
It don't work in cjs